Misc/NEWS.d/next/Library/2024-11-09-15-59-51.gh-issue-126624.bN53Va.rst

```diff
@@ -0,0 +1 @@
+Expose error code ``XML_ERROR_NOT_STARTED`` of Expat >=2.6.4.
```

Comment:
It would be good to add context for what module it is:

```suggestion
Expose error code ``XML_ERROR_NOT_STARTED`` of Expat >=2.6.4 in :mod:`xml.parsers.expat`.
```

Reply:
@ZeroIntensity good point, but let me use `xml.parsers.expat.errors` instead. Push coming up…

Reply:
Pushed.